projects/plugins/jetpack/modules/google-fonts/current/class-jetpack-google-font-face.php

```diff
@@ -28,6 +28,7 @@ public function __construct() {
 
 		// Collect and print fonts in use
 		add_action( 'wp_head', array( $this, 'print_font_faces' ), 50 );
+		add_action( 'admin_head', array( $this, 'print_font_faces' ), 50 );
```

Comment:
It would be better to keep using the `admin_print_styles` hook. WDYT?

Reply:
Yap, that would make the most sense and I considered it. But I was concerned that it'd rollback the intended outcome from the https://github.com/Automattic/jetpack/blob/47a0f6903d08f429e17c6c1a6c7eb60d4e3309fb/projects/plugins/jetpack/modules/google-fonts/current/class-jetpack-google-font-face.php#L53 function, that is loading only the fonts that are in use instead of printing all the fonts. Do you think it's okay if it does? Then I can update my PR to remove the codes related`admin_print_styles` hook here instead

Reply:
I think we have to print ALL fonts in the page/post editor as people may change the font of the block there. However, I'd prefer to print them only in the editor (without iframe) if possible as those styles are too heavy and we don't need them outside the editor.

Reply:
@arthur791004 makes sense! I have added codes here https://github.com/Automattic/jetpack/pull/34776/commits/52f1527f6bc6e2208b4d8f5b1293c8a649a2814a to load fonts only in GB editor pages. I switched from `admin_init` hook to `current_screen` hook because otherwise the current screen related information were not loaded.

Reply:
It seems to print the font CSS twice when the editor is iframed.

![image](https://github.com/Automattic/jetpack/assets/13596067/a7f511d9-e56e-45f5-8117-a00fe0489502)